package.json

```diff
@@ -2,19 +2,34 @@
   "name": "@erc725/erc725.js",
   "version": "0.21.3",
   "description": "Library to interact with ERC725 smart contracts",
-  "main": "build/main/src/index.js",
-  "typings": "build/main/src/index.d.ts",
-  "module": "build/module/src/index.js",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    },
+    "./utils": {
+      "require": "./dist/utils.js",
+      "import": "./dist/utils.mjs",
+      "types": "./dist/utils.d.ts"
+    },
+    "./dist/utils": {
+      "require": "./dist/utils.js",
+      "import": "./dist/utils.mjs",
+      "types": "./dist/utils.d.ts"
+    },
+    "./schemas/": "./schemas/",
+    "./docs/": "./docs/"
+  },
   "files": [
-    "build",
+    "dist",
     "schemas",
     "docs"
   ],
   "scripts": {
-    "build": "run-p build:*",
-    "build:main": "tsc -p tsconfig.json",
-    "build:module": "tsc -p tsconfig.module.json",
-    "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' nyc --reporter=text --reporter=lcov mocha",
+    "build": "tsup",
+    "test": "cross-env TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' nyc --project=tsconfig.mocha.json --reporter=text --reporter=lcov mocha --config=.mocharc.json",
```

Comment:
Maybe instead of passing the option as a CLI flag, we can set it up in the `tsconfig.mocha.json`.

https://typestrong.org/ts-node/docs/configuration/#via-tsconfigjson-recommended
https://typestrong.org/ts-node/docs/options/#compileroptions

```json
{
  "extends": "./tsconfig.module",
  "include": ["src/**/*.test.ts", "test/*.ts"]

  // Most ts-node options can be specified here using their programmatic names.
  "ts-node": {
    
  "compilerOptions": {
        "module": "commonjs"
    }
  }
}
```

Reply:
Might be possible, this is how it was documented inside of some threads about mocha with the same problem. Since we're using nyc as well, I felt more comfortable make it an environment setting.

Reply:
That works. Switching to it.